account_invoice_alternate_payer/models/account_payment.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class AccountPayment(models.Model):
+    _inherit = "account.payment"
+
+    @api.model
+    def default_get(self, default_fields):
+        rec = super().default_get(default_fields)
+        active_ids = self._context.get("active_ids")
+        invoices = (
+            self.env["account.move"]
+            .browse(active_ids)
+            .filtered(lambda move: move.is_invoice(include_receipts=True))
+        )
+        if invoices[0].alternate_payer_id:
```

Comment:
```suggestion
        if invoices and invoices[0].alternate_payer_id:
```
I got following error, if I install it directly with a blank database (not if account already installed).
```
  File "/opt/odoo/custom/src/odoo/odoo/models.py", line 1792, in _add_missing_default_values
    defaults = self.default_get(list(missing_defaults))
  File "/opt/odoo/auto/addons/account_invoice_alternate_payer/models/account_payment.py", line 20, in default_get
    if invoices[0].alternate_payer_id:
  File "/opt/odoo/custom/src/odoo/odoo/models.py", line 5737, in __getitem__
    return self.browse((self._ids[key],))
IndexError: tuple index out of range
```

Reply:
Solved @kittiu Thanks for your review.